packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -300,6 +302,38 @@ export const defaultL2Network: ArbitrumNetwork = {
   }
 }
 
+export const defaultL3Network: ArbitrumNetwork = {
```

Comment:
Taken from `l2l3_network.json` from testnode build. Here's the file content if you wanted to confirm addresses:
```typescript
{
  "l1Network": {
    "blockTime": 10,
    "chainID": 412346,
    "explorerUrl": "",
    "isCustom": true,
    "name": "EthLocal",
    "partnerChainIDs": [
      333333
    ],
    "isArbitrum": false
  },
  "l2Network": {
    "chainID": 333333,
    "confirmPeriodBlocks": 20,
    "ethBridge": {
      "bridge": "0xA584795e24628D9c067A6480b033C9E96281fcA3",
      "inbox": "0xDcA690902d3154886Ec259308258D10EA5450996",
      "outbox": "0xda243bD61B011024FC923164db75Dde198AC6175",
      "rollup": "0xf9B0F86aCc3e42B7DF373c9a8adb2803BF0a7662",
      "sequencerInbox": "0x16c54EE2015CD824415c2077F4103f444E00A8cb"
    },
    "explorerUrl": "",
    "isArbitrum": true,
    "isCustom": true,
    "name": "ArbLocal",
    "partnerChainID": 412346,
    "retryableLifetimeSeconds": 604800,
    "nitroGenesisBlock": 0,
    "nitroGenesisL1Block": 0,
    "depositTimeout": 900000,
    "tokenBridge": {
      "l1CustomGateway": "0xA191D519260A06b32f8D04c84b9F457B8Caa0514",
      "l1ERC20Gateway": "0x6B0805Fc6e275ef66a0901D0CE68805631E271e5",
      "l1GatewayRouter": "0xfE03DBdf7A126994dBd749631D7fbaB58C618c58",
      "l1MultiCall": "0x20a3627Dcc53756E38aE3F92717DE9B23617b422",
      "l1ProxyAdmin": "0x1A61102c26ad3f64bA715B444C93388491fd8E68",
      "l1Weth": "0xA1abD387192e3bb4e84D3109181F9f005aBaF5CA",
      "l1WethGateway": "0x77603b0ea6a797C74Fa9ef11b5BdE04A4E03D550",
      "l2CustomGateway": "0xD4816AeF8f85A3C1E01Cd071a81daD4fa941625f",
      "l2ERC20Gateway": "0xaa7d51aFFEeB32d99b1CB2fd6d81D7adA4a896e8",
      "l2GatewayRouter": "0x8B6BC759226f8Fe687c8aD8Cc0DbF85E095e9297",
      "l2Multicall": "0x052B15c8Ff0544287AE689C4F2FC53A3905d7Db3",
      "l2ProxyAdmin": "0x36C56eC2CF3a3f53db9F01d0A5Ae84b36fb0A1e2",
      "l2Weth": "0x582a8dBc77f665dF2c49Ce0a138978e9267dd968",
      "l2WethGateway": "0xA6AB233B3c7bfd0399834897b5073974A3D467e2"
    }
  },
  "l1TokenBridgeCreator": "0x38F35Af53bF913c439eaB06A367e09D6eb253492",
  "retryableSender": "0x7969b1d5beA6ac112665cf47806032BD3cf56eD3"
}
```